ci/cron/src/Main.hs

```diff
@@ -349,6 +349,10 @@ push_to_gcp gcp_credentials bash_lib local_path remote_path = do
         "gcs \"$GCRED\" cp \"" <> local_path <> "\" \"" <> remote_path <> "\"",
         "'"]
 
+check_files_match :: String -> String -> IO Bool
+check_files_match f1 f2 = do
+    fmap read (shell $ "if diff " <> f1 <> " " <> f2 <> " >/dev/null 2>&1; then echo True; else echo False; fi")
```

Comment:
Can't you use the exit code? I think something like this will work:

```suggestion
    (exitCode, stdout, stderr) <- readProcessWithExitCode "diff" [f1, f2] ""
    case exitCode of
      ExitSuccess -> return True
      ExitFailure 1 -> return False
      ExitFailure _ -> fail $ "Diff failed.\n" ++ "STDOUT:\n" ++ stdout ++ "\nSTDERR:\n" ++ stderr
```

Probably wanna capture that STDOUT and STDERR for the failure to be safe.

Reply:
Good point. Will add that once I get things working.